src/Components/AtbashIntro/AtbashIntro.js

```diff
@@ -1,57 +1,77 @@
+import Anime from 'react-anime';
 import './atbashIntro.css'
+import { clickMe, partOne, partTwo, partThree} from './AtbashIntroSections.js'
 
-function AtbashIntro() {
-    return (
-        <section className="container mb-6">
-            <div className="columns is-variable is-5">
-                <div className="column is-one-third">
-                    <img src={hieroglyph1} alt="ancient hieroglyphs depicting birds and other things" />
-                </div>
-                <div className="column">
-                    <div className="content"> 
-                        <p className="is-size-5">
-                            Cryptography is ancient, and has existed as long as people have wanted to 
-                            communicate in secret.
-                        </p>
-                        <p className="is-size-5">
-                            One of the oldest ciphers was designed before the English alphabet, but we can 
-                            easily translate it over: the Atbash cipher
-                        </p>
-                    </div>
-                </div>
-            </div>
-            <div className="column is-one-third">
-                <img src={hieroglyph2} alt="more ancient hieroglyphs" className="position-img" />
-            </div>
-            <div className="columns">
-                <div className="column is-two-thirds">
-                    <div className="content">
-                        <p className="is-size-5">
-                            Originally made for Hebrew, people would replace the first letter, aleph, with 
-                            the last letter tav. The second letter, bet, would be replaced with the 
-                            second-to-last letter, shin...
-                        </p>
-                        <p className="is-size-5">
-                            Can you guess how these letters would match up in an 
-                            English Atbash cipher?
-                        </p>
-                    </div>
+class AtbashIntro extends React.Component {
+    constructor(props) {
+        super(props);
+        this.timer = 0;
+        this.timers = []
+        this.state = {
+            clicked: [true, false, false],
+            finished: [true, false, false],
+            show: [true, false, false]
+        }
+    }
+
+    componentDidUnmount() {
+        this.timer = clearTimeout()
+    }
+
+    handleClickAt = (i) => {
+        if (this.state.clicked[i-1] === true) 
+            return
+        let newClick = Array.from(this.state.clicked)
+        newClick.splice(i-1, 1, true)
+        this.setState({
+            clicked: newClick
+        })
+    }
+
+    showContent = (i) => {
+        let newshow = Array.from(this.state.show)
+        newshow.splice(i-1, 1, true)
+        this.setState({
+            show: newshow
+        })
+        this.timers.push(setTimeout(() => {this.finishAnime(i)}, 1000)) 

```

Comment:
There's a lot of things going on here with timing your animations. Something to chew on that might make your life easier: rather than tracking the state of your three animations in three separate arrays, and working with manual timeouts, you could instead play around with [cascading lists](https://alain.xyz/libraries/react-anime#cascading-lists) in `react-anime`.

Reply:
I was totally fixated on cascading lists as timer based, so thanks for making me take another look. Gave it a different whirl in 53a95d1. Let me know what you think.